### DISCUSSION TITLE
Can't open web page

### DISCUSSION THREAD

**Discussion by hescluke** [open-webui/open-webui#2284] [General]
# Bug Report

## Description

**Bug Summary:**

I followed the commands at the bottom of doc:https://docs.openwebui.com/getting-started/ . The ollama serve can start successfully. When I run “.\start_windows.bat”, the terminal showed informations correctly, but the web page stucked at the beginning page. I can't even find any error logs, because everything from the terminal seems fine.

## Environment

windows10 
My computer cannot install docker, so I followed the instructions at the doc above. I met the same bug when I installed it on my lwindows, still stucking at the beginning page without any ERROR from the terminal.

node -v: v22.0.0
npm -v: 10.5.1
python -v: Python 3.11.9

## Logs and Screenshots

**stucked web page**
![image](https://github.com/open-webui/open-webui/assets/86816786/067368d1-0095-4e62-8730-de72d0e0afce)

**terminal outputs**
![image](https://github.com/open-webui/open-webui/assets/86816786/3d60db75-f68b-4a59-bbc4-92c7e4a3d028)

**Comment by elgarty** [comment]
Cant you just use Docker?

**Reply by hescluke** [reply]
Due to some policies, Docker is banned on this computer.

**Comment by MichaelFomenko** [comment]
you need first compile the code, before you can run it.

**Comment by justinh-rahb** [comment]
> you need first compile the code, before you can run it.

Yes, my guess is the OP forgot the `npm run build` step.

**Reply by hescluke** [reply]
I already did "npm run build", but still got this problem. An interesting thing is that, I follow the same steps from your docs on my another windows computer, it run correctly. But on this win, I did the exectly same steps, but the web page just stucked. Both two wins have the same version of node, npm and python. I even tried to copy the "open webui" folder from the normal win to this error win, and run "start_windows.bat", but the web page still got stucked

**Reply by hescluke** [reply]
All the files in the "open webui" folder are as follows:
`
Mode                 LastWriteTime         Length Name                                                                 
----                 -------------         ------ ----                                                                 
d-----         2024/5/15     15:13                .github                                                              
d-----         2024/5/15     15:14                .svelte-kit                                                          
d-----         2024/5/15     16:11                backend                                                              
d-----         2024/5/15     15:14                build                                                                
d-----         2024/5/15     15:13                cypress                                                              
d-----         2024/5/15     15:13                docs                                                                 
d-----         2024/5/15     15:13                kubernetes                                                           
d-----         2024/5/15     15:14                node_modules                                                         
d-----         2024/5/15     15:13                src                                                                  
d-----         2024/5/15     15:13                static                                                               
-a----         2024/5/15     15:13            214 .dockerignore                                                        
-a----         2024/5/15     15:13            480 .env                                                                 
-a----         2024/5/15     15:13            480 .env.example                                                         
-a----         2024/5/15     15:13            173 .eslintignore                                                        
-a----         2024/5/15     15:13            606 .eslintrc.cjs                                                        
-a----         2024/5/15     15:13             16 .gitattributes                                                       
-a----         2024/5/15     15:13           5612 .gitignore                                                           
-a----         2024/5/15     15:13             20 .npmrc                                                               
-a----         2024/5/15     15:13            212 .prettierignore                                                      
-a----         2024/5/15     15:13            242 .prettierrc                                                          
-a----         2024/5/16     22:57              0 a.txt                                                                
-a----         2024/5/15     15:13         124169 bun.lockb                                                            
-a----         2024/5/15     15:13           1806 Caddyfile.localhost                                                  
-a----         2024/5/15     15:13          22382 CHANGELOG.md                                                         
-a----         2024/5/15     15:13            356 confirm_remove.sh                                                    
-a----         2024/5/15     15:13            143 cypress.config.ts                                                    
-a----         2024/5/15     15:13        5192022 demo.gif                                                             
-a----         2024/5/15     15:13            229 docker-compose.amdgpu.yaml                                           
-a----         2024/5/15     15:13            148 docker-compose.api.yaml                                              
-a----         2024/5/15     15:13            110 docker-compose.data.yaml                                             
-a----         2024/5/15     15:13            280 docker-compose.gpu.yaml                                              
-a----         2024/5/15     15:13            823 docker-compose.yaml                                                  
-a----         2024/5/15     15:13           5496 Dockerfile                                                           
-a----         2024/5/15     15:13            927 i18next-parser.config.ts                                             
-a----         2024/5/15     15:13            676 INSTALLATION.md                                                      
-a----         2024/5/15     15:13           1099 LICENSE                                                              
-a----         2024/5/15     15:13            676 Makefile                                                             
-a----         2024/5/15     15:14         268129 package-lock.json                                                    
-a----         2024/5/15     15:13           2149 package.json                                                         
-a----         2024/5/15     15:13             79 postcss.config.js                                                    
-a----         2024/5/15     15:13          16502 README.md                                                            
-a----         2024/5/15     15:13           7442 run-compose.sh                                                       
-a----         2024/5/15     15:13            412 run-ollama-docker.sh                                                 
-a----         2024/5/15     15:13            471 run.sh                                                               
-a----         2024/5/15     15:13            900 svelte.config.js                                                     
-a----         2024/5/15     15:13            754 tailwind.config.js                                                   
-a----         2024/5/15     15:13           2750 TROUBLESHOOTING.md                                                   
-a----         2024/5/15     15:13            549 tsconfig.json                                                        
-a----         2024/5/15     15:13            270 update_ollama_models.sh                                              
-a----         2024/5/15     15:13            231 vite.config.ts                                                       


 `

**Comment by cattoman1** [comment]
im having the same Problem But i did the `npm run build` step

**Reply by hescluke** [reply]
Do you have any solution yet?

**Reply by cattoman1** [reply]
no I also went to ask in the discord i didnt get any answer
